.dict_custom.txt

```diff
@@ -99,3 +100,7 @@ unhandled
 unallocated
 mypy
 destructor
+destructor's
+VALGRIND
```

Comment:
According to their website this should simply be `Valgrind`

Reply:
Fixed https://github.com/pyccel/pyccel/pull/1580/commits/2760eebc29beb59adec461e85472f699b981ffff.